drivers/vfs/driver.go

```diff
@@ -47,7 +48,7 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 		key = strings.ToLower(key)
 		switch key {
 		case "vfs.imagestore", ".imagestore":
-			d.homes = append(d.homes, strings.Split(val, ",")...)
+			d.additionalHomes = strings.Split(val, ",")
```

Comment:
Can the `imagestore` option be provided twice? This would discard the first value. (“That’s invalid and should be rejected” seems a perfectly fine answer to me. “That’s undocumented but we use all of them” constraints us long-term but works fine. Silently discarding data seems not ideal.)

Reply:
good observation, I've restored the previous code